extensions/amp-story/0.1/amp-story.js

```diff
@@ -1088,23 +989,10 @@ export class AmpStory extends AMP.BaseElement {
     if (fallbackEl) {
       this.toggleFallback(true);
     } else {
-      this.buildUnsupportedBrowserOverlay_();
+      this.storeService_.dispatch(Action.TOGGLE_SUPPORTED_BROWSER, false);
```

Comment:
Nit: This naming is slightly weird.  Technically, even in the other case (`if (fallbackEl)`), the browser is not supported by `amp-story`, but the publisher has specified their own fallback content to show instead of our default.  But, in that case, the `SUPPORTED_BROWSER_STATE` will be true, even though the browser is not supported.

Two possible fixes:

1. Move the `fallbackEl` logic above into `UnsupportedBrowserLayer`
2. Rename `SUPPORTED_BROWSER_STATE` and `TOGGLE_SUPPORTED_BROWSER` to something that clearly indicates you're referring to the state of the default stories supported browser UI, rather than whether or not the browser is supported.

Reply:
Great idea, thanks, that was super confusing.
I went with option 1, but kept the fallback logic in the amp-story file so we can keep using the ``this.getFallback`` and ``this.toggleFallback``methods.
Let me know if you had something else in mind!